packages/form-js-viewer/assets/form-js.css

```diff
@@ -36,8 +37,8 @@
   --color-borders: var(--color-grey-225-10-55);
   --color-borders-disabled: var(--color-grey-225-10-75);
   --color-warning: var(--color-red-360-100-45);
-  --color-accent: var(--color-blue-205-100-45);
-  --color-accent-dark: var(--color-blue-205-100-45);
+  --color-accent: var(--color-blue-205-100-40);
```

Comment:
Accent and accent dark have the same shade, is this intentional? If not we should merge the tag.

Reply:
It was the same before, not sure whether it's intentional though. From the naming I assume it was meant to be `darker`, so maybe just let's use `--color-accent-dark: var(--color-blue-205-100-35);`

_Not focussed_
![image](https://user-images.githubusercontent.com/9433996/198323549-92eee266-f854-4c39-905c-0150b20d5b51.png)

_Focussed_
![image](https://user-images.githubusercontent.com/9433996/198323442-a442b184-3c3f-4eac-aaac-69c64fcd7831.png)

Reply:
Alternative: remove the border completely and keep the browser default on focus for buttons.

Reply:
Just found out that we are setting the border-color to prevent using the default `--color-borders` for other inputs, which is a grey tone. 

Let's simply remove the `--color-accent-dark` which doesn't have any value and simply use `--color-accent` to keep things as they are.

Cf. https://github.com/bpmn-io/form-js/pull/376/commits/e45eba75d7705fbe5cdf50e251c45339d3fc4019